specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2018-06-01-preview/locations.json

```diff
@@ -0,0 +1,140 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "HDInsightManagementClient",
+        "description": "The HDInsight Management Client.",
+        "version": "2015-03-01-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "security": [{
+        "azure_auth": [
+            "user_impersonation"
+        ]
+    }],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
+        "/subscriptions/{subscriptionId}/providers/Microsoft.HDInsight/locations/{location}/usages": {
+            "get": {
+                "tags": [
+                    "Regions"
+                ],
+                "operationId": "Location_ListUsages",
+                "description": "Lists the usages for the specified location.",

```

Comment:
I think this operation should be named as `Usages_ListByLocation` and the entire swagger should be renamed to "Usages.json" instead of Locations. Are you planning to add more operations in the nearest future for Location? e.g. put/get/delete ? if not then it does note make sense to me to have Location swagger that actually provides API for a child resource/operation of the Location.

Reply:
We're most likely going to add more operations here.